src/averageAges.js

```diff
@@ -37,7 +39,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filterWomen = !withChildren
+    ? people.filter(person => person.sex === 'f')
+    : people.filter(person =>
+      people.find(child => child.mother === person.name));
+
+  return filterWomen.reduce((sum, per) =>
+    sum + (per.died - per.born), 0) / filterWomen.length;
```

Comment:
```suggestion
  const filterWomen = !withChildren
    ? people.filter(person => person.sex === 'f')
    : people.filter(person => (
      people.find(child => child.mother === person.name)));

  return filterWomen.reduce((sum, per) => (
    sum + (per.died - per.born)), 0) / filterWomen.length;
```

examples how to put your brackets,  you can read it again in checklist